### PR TITLE
v0.97.0 - colour util

### PR DESCRIPTION
**Pull request by sonnyprince**
_Addition of colour utility class._

Addition of `.u-color-secondary`. It is the same color as `.u-color-link` but i thought it would be better for items that aren't a link but require that colour.

### CODE REVIEW

**Comment by kevinrodrigues** [comment]
It's probably better to open these type of prs after the previous one merges.

**Comment by sonnyprince** [comment]
@kevinrodrigues will do in the future 👍

File: src/scss/trumps/_utilities.scss [hunk 1]
```diff
@@ -139,7 +139,8 @@
     color: $color-text;
 }
 
-.u-color-link {
+.u-color-link,
+.u-color-secondary {
```
Comment (markedup) [inline src/scss/trumps/_utilities.scss]: Without a pre-existing `.u-color-primary` class, I'm not sure I understand what this is for, or how we should apply it in future; is there a better name for this?

Comment (sonnyprince) [reply src/scss/trumps/_utilities.scss]: `-secondary` refers to directly to a colour variable within fozzie - as does `-link` ($color-link-default and $color-secondary (both use $blue)).

If the classes were not directly linked to a fozzie colour var I would agree :)

Comment (markedup) [reply src/scss/trumps/_utilities.scss]: Okay, that connection isn't obvious from this instance, so I was concerned it wouldn't be obvious for other developers. I'll approve in this instance though.

**Comment by sonnyprince** [comment]
@ashleynolan It would yes, but i was just trying to make non-link html not have CSS that refers to links

**Pull request merged**